plugin/storage/integration/grpc_test.go

```diff
@@ -87,6 +140,31 @@ func (s *GRPCStorageIntegrationTestSuite) cleanUp() error {
 	return s.initialize()
 }
 
+type memoryStorePlugin struct {
```

Comment:
do you need this type in the test, or can you use one from plugin/storage/grpc/memory/plugin.go ?